lib/components/app-service-bot.js

```diff
@@ -5,11 +5,17 @@
  * @constructor
  * @param {MatrixClient} client The client instance configured for the AS bot.
  * @param {AppServiceRegistration} registration The registration that the bot
+ * @param {Function=} setJoined Called when a /joined_members request has a remote user.
```

Comment:
You put this in the middle of the previous line's explanation :p

Reply:
Actually, it was already like that :(